Wikipedia/View Controllers/RecentSearches/RecentSearchesViewController.m

```diff
@@ -39,15 +34,16 @@ - (void)viewDidLoad {
     [self setupTable];
 
     [self updateTrashButtonEnabledState];
-
-    [self.table setBackgroundColor:[UIColor clearColor]];
-    self.table.backgroundView.backgroundColor = [UIColor clearColor];
-    self.table.rowHeight                      = cellHeight;
 }
 
 - (void)setupTable {
-    self.table.separatorStyle = UITableViewCellSeparatorStyleNone;
     [self.table registerNib:[UINib nibWithNibName:@"RecentSearchCell" bundle:nil] forCellReuseIdentifier:@"RecentSearchCell"];
+
+    self.table.estimatedRowHeight = 52.f;
+    self.table.rowHeight          = UITableViewAutomaticDimension;
+
+    [self.table setBackgroundView:nil];
```

Comment:
Used grouped table layout to get, for free, separators above the first cell and below the last cell, but grouped layout adds a background which causes the translucency effect to break. Nil'ing out the background view fixes it.